Changelog.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 2.1.1 - 2021-01-22
```

Comment:
This would be a new feature and not a bug fix. Consider doing a minor version update instead of a patch version update

Reply:
Updated it to 2.2.0.